### DISCUSSION TITLE
Micro steps by pressing the knob

### DISCUSSION THREAD

**Discussion by jxer** [mriscoc/Ender3V2S1#181] [Ideas]
jyers has microstep up and microstep down in tuning for z offset.

is it possible to add this? I don't always remember which way is up when i am tuning on the fly.

i like to be able to tap the button and have it move up or done, so i can focus on the print and not trying to remember the z number.

**Comment by mriscoc** [comment]
We want to avoid the excesive wear out of the push button of the knob in the screen unit. The baby steps are done with the rotary encoder.

**Reply by jxer** [reply]
is this a common problem? i have never seen this mentioned. and micro steps is maybe 10 to 15 clicks. I don't really see that being excessive.

**Comment by northcoastam** [comment]
I quite like that jyers feature too - I found this conversation while searching to check if this firmware has the same option.  I would like to second this request.
